### ISSUE TITLE
Consider using expo-crypto for crypto instead

### ISSUE THREAD

**Issue by pcowgill** [parshap/node-libs-react-native#24]
Consider using `expo-crypto` for `crypto` instead of `react-native-crypto`

Note that the `expo-crypto` package works in non-Expo React Native apps too.

https://docs.expo.io/versions/latest/sdk/crypto/
https://www.npmjs.com/package/expo-crypto
https://github.com/tradle/react-native-crypto/issues/42

**Comment by parshap** [comment]
If you believe that's a better solution, I'd be happy to accept a PR.

**Comment by pcowgill** [comment]
> If you believe that's a better solution, I'd be happy to accept a PR.

Great! I'll test it a bit more locally and open a PR if that seems like the way to go.

FYI, the current `README` on the npm website for `expo-crypto` is slightly out of date and misleading as of today, until their next release. Here's an issue that has been resolved about that:
https://github.com/expo/expo/issues/6569

TL;DR: `expo-crypto` works in both "managed" and "bare" Expo apps as well as vanilla React Native apps.

**Comment by pcowgill** [comment]
I'm wondering about this line from the `expo-crypto` docs, though: "in an equivalent manner to the Node.js core crypto API"

It may have similar functionality but not attempt to polyfill the API 1:1. I'll keep researching.

**Comment by pcowgill** [comment]
`expo-crypto` doesn't have the same API as the node core `crypto` module, so it won't work for a polyfill solution, at least for now

https://expo.canny.io/feature-requests/p/nodejs-crypto-module-full-features
